terraform/aws/.terraform.lock.hcl

```diff
@@ -2,39 +2,44 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/hashicorp/aws" {
```

Comment:
Is this file needs to be updated?

Reply:
Yes, because we are updating terraform deps here as well.

Reply:
[Docs](https://developer.hashicorp.com/terraform/language/files/dependency-lock#lock-file-location): `You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.`